staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/celcoststability_test.go

```diff
@@ -2022,7 +2022,7 @@ func TestCelEstimatedCostStability(t *testing.T) {
 				"optional.of('a').hasValue()":                         2,
 				"optional.of('a').or(optional.of('a')).hasValue()":    4, // or() is short-circuited
 				"optional.none().or(optional.of('a')).hasValue()":     4,
-				"optional.of('a').optMap(v, v == 'value').hasValue()": 17,
+				"optional.of('a').optMap(v, v == 'value').hasValue()": 18,
```

Comment:
This is backward incompatible from a cost perspective.

Reply:
This change is due to a bug earlier which the basic cost of optMap is missed. It is more of a bug fix and only affect the estimated cost of option map/flatMap by adding 1 cost to each level. I'll add it into the PR description.